composer.json

```diff
@@ -67,16 +67,16 @@
         "padraic/phar-updater": "^1.0.4",
         "pimple/pimple": "^3.2",
         "sebastian/diff": "^1.4 || ^2.0 || ^3.0",
-        "symfony/console": "^3.2 || ^4.0",
-        "symfony/filesystem": "^3.2 || ^4.0",
+        "symfony/console": "~3.2.13|| ^3.3.6 || ^4.0",
```

Comment:
why do you bump up other symfony dependencies to 3.3 but `symfony/console` stay with 3.2? IMO it will be better to lock all dependencies with one version

Reply:
What if we get them all up to 3.4?